docs/source/_templates/breadcrumbs.html

```diff
@@ -0,0 +1,28 @@
+{%- extends "sphinx_rtd_theme/breadcrumbs.html" %}
```

Comment:
We extend [this file](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/breadcrumbs.html) to modify `Edit on GitHub` link for examples.